recipes/xlsxio/all/conanfile.py

```diff
@@ -0,0 +1,144 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout, CMakeDeps
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.54.0"
+
+
+class XlsxioConan(ConanFile):
+    name = "xlsxio"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/brechtsanders/xlsxio"
+    description = "Cross-platform C library for reading values from and writing values to .xlsx files."
+    topics = ("xlsx",)
+    license = "MIT"
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC":[True, False],
+        "shared": [True, False],
+        "build_pc_files": [True, False],
+        "build_tools": [True, False],
+        "build_examples": [True, False],
+        "with_libzip": [True, False],
+        "with_minizip_ng": [True, False],
+        "with_wide": [True, False],
+        "build_documentation": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+        "build_pc_files": True,
+        "build_tools": True,
```

Comment:
Do you know why these options are enabled? Does it make sense when using Conan  (as conan generates pc files if you chose the generator for so)?

Are build tools required here as well? What do they do?

Reply:
1.The pc file is generated from template.pc.in when build with cmake, when using Conan it do not make sense, i will remove it.
2.Here "build_tools" is not tools used to build project,  but is runtime target which can be deployed to a device.